Makefile

```diff
@@ -0,0 +1,50 @@
+.PHONY: all help init clean test lint format type-check coverage
```

Comment:
I'd prefer if you could remove the Makefile. Thanks for putting in the effort though!
I do see your point that it's difficult for contributors to know what they should run. I'll work on improving that 👍